introduction.md

```diff
@@ -1 +1,3 @@
 Fernando Luiz Goulart is a Brazilian developer, team leader and Scrum Master favorite course Javascript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?